src/Controller/OverviewController.php

```diff
@@ -99,6 +99,34 @@ public function show_all(): Response
 
   }
 
+  /**
+   * Basically the same page as /overview, except shows all announcements of
+   * the currently logged in user.
+   * Future Additions - allow users to modify announcements
+   * 
+   * @author Daniel Boling
+   * @return rendered overview.html.twig
+   * 
+   * @Route("/overview/user", name="show_all_user")
+   */
+  public function show_user(): Response
+  {
+
+    $date = getdate();
+    $user = $this->getUser();
+
+    $announcement = $this->getDoctrine()
+      ->getRepository(Announcement::class)
+      ->findByUser($user)
+      ;
```

Comment:
You could also do `$user->getAnnouncements()` for this which might be easier or more understandable.

Reply:
I did it with the query for the future in case we might want/need to change something or add something. I can do it your way if you think it would be a better idea though.